server/datastore/inmem/hosts.go

```diff
@@ -103,6 +104,33 @@ func (d *Datastore) ListHosts(opt kolide.HostListOptions) ([]*kolide.Host, error
 	low, high := d.getLimitOffsetSliceBounds(opt.ListOptions, len(hosts))
 	hosts = hosts[low:high]
 
+	// Filter additional info
```

Comment:
In the future please feel free not to implement anything in inmem. It's been mostly removed from use and I need to finish tearing it out.

Reply:
Added tests, but did you also want me to remove this inmem implementation?

Reply:
No need for that. I'll be removing all of this soon in any case.